Feature 1159 discussions (#1634)
dtcenter/METplus: docs/Contributors_Guide/user_support.rst

************
User Support
************

Support for the METplus components is provided through the
`METplus GitHub Discussions Forum <https://github.com/dtcenter/METplus/discussions>`_.
Discussions is a collaborative communication forum, where the METplus
community can ask and answer questions, gather feedback, and share tips and
tricks. Discussions are categorized to help community members find related
discussions and begin conversations in the right place. 

Support Responsibilities
========================

> Five staff members take turns monitoring Discussions each day of the week.

Reviewer: ```suggestion
Five staff members take turns monitoring Discussions each day of the week. The responsibilities for each daily assignee are described below.
```

Review the Tips/Tricks Page
---------------------------

Review the `Tips/Tricks <https://github.com/dtcenter/METplus/discussions/categories/tips-tricks>`_
page.

The Tips/Tricks page is for discussion only and does not have the ability to
have an *answer* assigned to it. Check for any new dicussions that have been
added by users that are actually questions. If there are, move the discussion

> to a more approprate category so that it can be found when filtering for

Reviewer: ```suggestion
to a more appropriate category so that it can be found when filtering for
```

unanswered discussions.  Please note that if the discussions have been
filtered by "Unanswered" and the Tips/Tricks category is selected, no results
will be found because these discussions are not answerable.

If a new tip or trick has been added, please review the addition.  Consider
adding any comments to add detail or clarify, and thank the user for their
contribution.

Review the Unanswered Discussions
---------------------------------

Review the `unanswered discussions <https://github.com/dtcenter/METplus/discussions?discussions_q=is%3Aunanswered>`_.

> * For new discussions that have been posted within 24 hours:

Reviewer: ```suggestion
* For new discussions that have been posted on the current day (i.e. since last midnight):
```

>   * If the issue is time sensitive, i.e. related to an upcoming training series

Reviewer: ```suggestion
  * If the issue is time sensitive (i.e. related to an upcoming training series
```

>     session, take action.  See :ref:`actions_new_discussions`.

Reviewer: ```suggestion
    session) or from a user with priority support, take action.  See :ref:`actions_new_discussions`.
```
